src/redis_cmd.cc

```diff
@@ -788,11 +788,16 @@ class CommandSetBit : public Commander {
 class CommandBitCount : public Commander {
  public:
   Status Parse(const std::vector<std::string> &args) override {
-    try {
-      if (args.size() >= 3) start_ = std::stoi(args[2]);
-      if (args.size() >= 4) stop_ = std::stoi(args[3]);
-    } catch (std::exception &e) {
-      return Status(Status::RedisParseErr, errValueNotInterger);
+    if (args.size() == 3) {
+      return Status(Status::RedisParseErr, errInvalidSyntax);
+    }
+    if (args.size() == 4) {
+      try {
+        start_ = std::stoi(args[2]);
```

Comment:
How about changing the start and stop to int64 as well? since the newer Redis version can support byte|bit mode.

Reply:
What is byte|bit mode? I didn't find it, can you give me some references?

Reply:
Can see the doc: https://redis.io/commands/bitcount

the last parameter can be bit or byte.

Reply:
> Redis version >= 7.0.0: Added the `BYTE|BIT` option.

Thank you, I just learned that.

Reply:
Did you just want to replace it with int64, or did we implement this option too? Redis 7.0.0 doesn't seem to be released.

Reply:
yes, just replace them with int64, to prevent suffering the same problem when we want to implement the option.

Reply:
> BITPOS key bit [start [end [BYTE|BIT]]]

OK, this part as well.

Reply:
cool~